modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -3252,6 +3253,13 @@ def iloc_mut(partition, row_internal_indices, col_internal_indices, item):
         return self.__constructor__(new_modin_frame)
 
     def sort_rows_by_column_values(self, columns, ascending=True, **kwargs):
+        # Our algebra sort is only implemented for Engines that support virtual partitioning.
+        if Engine.get() in ["Ray", "Dask"]:
```

Comment:
```suggestion
        if Engine.get() in ["Ray", "Dask", "Unidist"]:
```
Unidist engine also supports virtual partitioning.

Reply:
It seems to me that unidist failed because you didn't make the respective changes for pandas on unidist as for pandas on ray/dask. Please add the changes to pandas on unidist too.

Reply:
Added!